### PR TITLE
EveOnline::ESI::CharacterIndustryJob, EveOnline::ESI::CorporationIndustryJob

### PR DESCRIPTION
**Pull request by Mekaret**
Hi,
My commits for Eveonline::ESI::Industry::CharacterJobs and Eveonline::ESI::Industry::CorporationJobs.
RSPEC parts coming soon.

@+
Meka

### CODE REVIEW

File: lib/eve_online/esi/industry_character_jobs.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+module EveOnline
+  module ESI
+    class IndustryCharacterJobs < Base
```
Comment (biow0lf) [inline lib/eve_online/esi/industry_character_jobs.rb]: Rename to CharacterIndustryJob.

File: lib/eve_online/esi/industry_character_jobs.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+      def initialize(options)
+        super
+
+        @character_id = options[:character_id]
+      end
+
+      def get_industry_jobs
+        output = []
+        response.each do |asset|
```
Comment (biow0lf) [inline lib/eve_online/esi/industry_character_jobs.rb]: asset -> job

File: lib/eve_online/esi/industry_corporation_jobs.rb [hunk 3]
```diff
@@ -0,0 +1,32 @@
+module EveOnline
+  module ESI
+    class IndustryCorporationJobs < Base
```
Comment (biow0lf) [inline lib/eve_online/esi/industry_corporation_jobs.rb]: Rename to CorporationIndustryJob

File: lib/eve_online/esi/industry_corporation_jobs.rb [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+      def initialize(options)
+        super
+
+        @corporation_id = options[:corporation_id]
+      end
+
+      def get_industry_jobs
+        output = []
+        response.each do |asset|
```
Comment (biow0lf) [inline lib/eve_online/esi/industry_corporation_jobs.rb]: asset -> job

File: lib/eve_online/esi/industry_character_jobs.rb [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+      attr_reader :character_id
+
+      def initialize(options)
+        super
+
+        @character_id = options[:character_id]
+      end
+
+      def get_industry_jobs
```
Comment (biow0lf) [inline lib/eve_online/esi/industry_character_jobs.rb]: get_industry_jobs -> jobs

File: lib/eve_online/esi/industry_corporation_jobs.rb [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+      attr_reader :corporation_id
+
+      def initialize(options)
+        super
+
+        @corporation_id = options[:corporation_id]
+      end
+
+      def get_industry_jobs
```
Comment (biow0lf) [inline lib/eve_online/esi/industry_corporation_jobs.rb]: get_industry_jobs -> jobs

File: lib/eve_online/esi/models/industry_jobs.rb [hunk 7]
```diff
@@ -0,0 +1,123 @@
+module EveOnline
+  module ESI
+    module Models
+      class IndustryJobs < Base
```
Comment (biow0lf) [inline lib/eve_online/esi/models/industry_jobs.rb]: Rename to IndustryJob

File: lib/eve_online/esi/models/industry_jobs.rb [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+        #   output_location_id (integer): Location ID of the location to which the output of the job will be delivered. Normally a station ID, but can also be a corporation facility ,
+        #   pause_date (string, optional): Date and time when this job was paused (i.e. time when the facility where this job was installed went offline) ,
+        #   probability (number, optional): Chance of success for invention ,
+        #   product_type_id (integer, optional): Type ID of product (manufactured, copied or invented) ,
+        #   runs (integer): Number of runs for a manufacturing job, or number of copies to make for a blueprint copy ,
+        #   start_date (string): Date and time when this job started ,
+        #   station_id (integer): ID of the station where industry facility is located ,
+        #   status (string): status string = ['active', 'paused', 'ready', 'delivered', 'cancelled', 'reverted'],
+        #   successful_runs (integer, optional): Number of successful runs for this job. Equal to runs unless this is an invention job
+        # }
```
Comment (biow0lf) [inline lib/eve_online/esi/models/industry_jobs.rb]: Drop this comment

File: lib/eve_online/esi/models/industry_jobs.rb [hunk 9]
```diff
@@ -0,0 +1,123 @@
...
+        def as_json
+          {
+            activity_id: activity_id,
+            blueprint_id: blueprint_id,
+            blueprint_location_id: blueprint_location_id,
+            blueprint_type_id: blueprint_type_id,
```
Comment (biow0lf) [inline lib/eve_online/esi/models/industry_jobs.rb]: According Swagger spec for ESI, you miss `completed_character_id` and `completed_date` here.

File: lib/eve_online/esi/models/industry_jobs.rb [hunk 10]
```diff
@@ -0,0 +1,123 @@
...
+            cost: cost,
+            duration: duration,
+            end_date: end_date,
+            facility_id: facility_id,
+            installer_id: installer_id,
+            job_id: job_id,
+            licensed_runs: licensed_runs,
+            output_location_id: output_location_id,
```
Comment (biow0lf) [inline lib/eve_online/esi/models/industry_jobs.rb]: You miss `pause_date`, `probability` and `product_type_id` here.

File: lib/eve_online/esi/models/industry_jobs.rb [hunk 11]
```diff
@@ -0,0 +1,123 @@
...
+            runs: runs,
+            start_date: start_date,
+            station_id: station_id,
+            status: status
```
Comment (biow0lf) [inline lib/eve_online/esi/models/industry_jobs.rb]: You miss `successful_runs` here.

**Comment by biow0lf** [comment]
Generally, I like changes. Just fix code review issues, so I can merge changes.

If you are not familiar with `rspec`, just skip specs. I will write them later.

Also, can you provide example response for this requests?

File: README.md [hunk 12]
```diff
@@ -1304,8 +1304,26 @@ character_portrait.huge # => "http://image.eveonline.com/Character/90729314_512.
 #### Industry
 
 ##### List character industry jobs
+```ruby
+options = { token: 'token123', character_id: 90729314 }
+
```
Comment (biow0lf) [inline README.md]: Documentation also need some love :)

Comment (Mekaret) [reply README.md]: Done ;-)
I love the way you code...

Perhaps I should rename the names of files too ?

Comment (biow0lf) [reply README.md]: Rename files too. And make documentation looks like everywhere in project. E.g. https://github.com/biow0lf/eve_online#get-character-assets

**Comment by Mekaret** [comment]
Close this request plz, I make another one.
Now I use personal branch (industry) with fork, more easier to follow your repo.

**Pull request closed without merging**